src/main/java/com/java_essential/diledsys/homeWork/ClassRoom/Pupil.java

```diff
@@ -0,0 +1,9 @@
+package com.java_essential.diledsys.homeWork.ClassRoom;
+
+public abstract class Pupil {
```

Comment:
NOTE: If you want to describe the behavior of an object only, prefer interface usage